lib/miq_automation_engine/engine/miq_ae_object.rb

```diff
@@ -30,6 +30,10 @@ class MiqAeObject
     # Default conversion for Service Models
     SM_LOOKUP         = Hash.new { |_, k| k.classify }.merge(
       'ems'                    => 'ExtManagementSystem',
+      'vm'                     => 'Vm',
+      'user'                   => 'User',
+      'storage'                => 'Storage',
+      'host'                   => 'Host',
```

Comment:
@mkanoor Why do we need to add these extra values here?  They are already be handled by the Hash.new default block.